js/ccf-app/src/global.ts

```diff
@@ -253,8 +253,8 @@ export interface CCFConsensus {
   /**
    * Get the status of a transaction by ID, provided as a view+seqno pair.
    * This is a node-local property - while it will converge on all nodes in
-   * a healthy network, it is derived from distributed state rather than
-   * distributed itself.
+   * a healthy network, it is either derived from distributed state in BFT,
+   * or replicated out by the primary in CFT.
```

Comment:
Shouldn't this be a copy-paste of the full C++ docstring?

Reply:
Yes, it will be in a moment, since we agree on the phrasing now.